ceph.spec.in

```diff
@@ -22,7 +22,13 @@
 # bcond syntax!
 #################################################################################
 %bcond_without ocf
+# dependencies for running "make check"
 %bcond_with make_check
+# runtime dependencies for testing ceph. please note, some of them are covered
+# by `make_check` dependencies. they are listed as BuildRequires simply because
```

Comment:
nit: please use `'` or `"` instead of this funny "accent grave" character which is not English at all

Reply:
fixed.